ui/raidboss/data/03-hw/dungeon/the_vault.js

```diff
@@ -123,10 +84,7 @@
       condition: function(data) {
         return data.CanStun();
       },
-      infoText: {
-        en: 'Interrupt the Knight',
-        fr: 'Interrompez le chevalier',
-      },
+      response: Responses.interupt(),
```

Comment:
This loses the thing to interrupt?

Reply:
And shouldn't this be stun and not interrupt?

Reply:
i think this will be a generall question how to deal with stun/interupts on targets (due to the languages)

should stun/interupt just be used for bosses? and adds will not have response lines?

Reply:
is there a way to access the "source" from the trigger inside a response? Because than you could just relay on the Source itself (to call out stun target in all languages)

Reply:
Yes, matches.source should have what you want.